src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -13,6 +13,8 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
 
+import hudson.slaves.ComputerLauncher;
+import io.fabric8.kubernetes.client.utils.Serialization;
```

Comment:
Now unused?

Reply:
damn

Reply:
:man_shrugging: At some point maybe we should clean up imports etc.